client/ting.jsx

```diff
@@ -59,32 +65,43 @@ const Ting = React.createClass({
         });
 
         this._socket.on('message', (data) => {
-            this.refs.history.onMessage(data);
+            if (!this.state.active && data.username != this.state.intendedUsername) {
+                this.setState({
```

Comment:
So do you suggest to add every 'state' variable into redux store even if that variable is used only from one specific file ?